### PR TITLE
Added instructions for Mac OS.

### PR DESCRIPTION
**Pull request by EthanCentaurai**

### CODE REVIEW

**Comment by xPaw** [comment]
Is the curl enabled by default?

File: README.md [hunk 1]
```diff
@@ -19,6 +21,12 @@
 2. Extract the contents of this script to the same folder
 3. Run the script: `php cheat.php`
 
+### Mac
+
+0. (optional) Launch the App Store and download any updates for macOS. Newer versions of macOS have php and curl included by default.
+1. Extract the contents of this script to the Downloads folder.
+2. Launch Terminal and run the script: `php downloads/cheat.php`
```
Comment (rossengeorgiev) [inline README.md]: What about python?

**Comment by EthanCentaurai** [comment]
@xPaw It seems so yes. I've been able to run the php script on a fresh install of High Sierra without issues so far.

@rossengeorgiev Python 2.7.10 is included by default on High Sierra. I have not tested the script yet but chances are users will need to download a newer version.

**Comment by rossengeorgiev** [comment]
2.7.10 is not that old. Try it

**Comment by EthanCentaurai** [comment]
I had to `sudo easy_install pip` prior to `pip install requests tqdm` but otherwise everything appears to be ok so far.  I might have to add that to the readme.

File: README.md [hunk 2]
```diff
@@ -27,10 +35,20 @@ php cheat.php token2
 
 ## Python
 
+### Windows
```
Comment (rossengeorgiev) [inline README.md]: Should really be Linux/Cygwin

**Pull request merged**